packages/admin-page/src/header.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * A logo for Jetpack.
+ *
+ * @package jetpack-logo
+ */
+
+namespace Automattic\Jetpack\AdminPage;
+
+use Automattic\Jetpack\Assets\Logo;
+
+/**
+ * Create and Render the Jetpack Header
+ */
+class Header {
+
+	/**
+	 * Build and render the jetpack page Header
+	 *
+	 * @return string
+	 */
+	public function render() {
+		$jetpack_admin_url = admin_url( 'admin.php?page=jetpack' );
+
+		$logo = new Logo();
+		$logo = apply_filters( 'jetpack-admin-page-header-logo', $logo );
```

Comment:
Is this filter necessary? If it is, should it have underscores instead of dashes?

Reply:
Seems like this was just a brainfart after switching between BEM CSS classes and WP coding conventions. I'll have a look.